src/components/inputs/Selector.tsx

```diff
@@ -21,13 +21,22 @@ export enum SelectorType {
   STATUS_SELECTOR = 'statusSelector',
   TEMPLATE_SELECTOR = 'templateSelected',
 }
+type SelectorOptionsType = {
+  [K in keyof typeof SelectorType as `${(typeof SelectorType)[K]}`]: K extends 'ASSIGNEE_SELECTOR'
```

Comment:
```suggestion
  [K in keyof typeof SelectorType as typeof SelectorType[K]]: K extends 'ASSIGNEE_SELECTOR'
```

I believe this should work as well. Can you please confirm

Reply:
It might also be a good idea to extract the type definition from `keyof` in a separate type / enum so we can simply do `[K in SelectorTypeKeys]`... but this is not required.